transaction-service/transaction-module/src/main/kotlin/com/ajaxsystems/infrastructure/grpc/service/TransactionGRPCService.kt

```diff
@@ -1,40 +1,34 @@
-package ua.com.foxminded.restClient.gprc
+package com.ajaxsystems.infrastructure.grpc.service
 
+import com.ajaxsystems.application.useCases.FindTransactionsInputPort
+import com.ajaxsystems.infrastructure.database.mapper.toListResponse
```

Comment:
and here your grpc infrastructure depends on database infrastructure, which is bad. see if your mapper does not violate SRP

Reply:
I agree. Changed